File: message_board/list.cs
using System;
using System.Collections.Generic;

class Message
{
    public string Name { get; set; }
    public string Content { get; set; }
}

class Program
{
    static List<Message> messages = new List<Message>();//Is static necessary?

    static void ShowMessages()
    {
        if (messages.Count == 0)
        {
            Console.WriteLine("No message yet.");
            return;
        }
        for (int i = 0; i < messages.Count; i++)
        {
            Console.WriteLine($"{i+1}. {messages[i].Name}: {messages[i].Content}");
        }
    }

    static void AddMessage()
    {
        Console.Write("Your name: ");
        string name = Console.ReadLine();
        Console.Write("Your message: ");
        string content = Console.ReadLine();
        messages.Add(new Message { Name = name, Content = content });
        Console.WriteLine("Message posted!");
    }

    static void Main(string[] args)
    {
        while (true)
        {
            Console.WriteLine("\n--- Message Board ---");
            Console.WriteLine("1. View messages");
            Console.WriteLine("2. Post message");
            Console.WriteLine("3. Exit");

            Console.Write("Choose: ");
            string choice = Console.ReadLine();

            if (choice == "1") {
                ShowMessages();
            } else if (choice == "2") {
                AddMessage();
            } else if (choice == "3") {
                break;
            } else {
                Console.WriteLine("Invalid choice");
            }
        }
    }
}

File: message_board/list.py
messages = []

def show_messages():
    if not messages:
        print("No messages yet.")
        return
    for i, msg in enumerate(messages):
        print(f"{i+1}. {msg['name']}: {msg['content']}")

def add_message():
    name = input("Your name: ")
    content = input("Your message: ")
    messages.append({"name": name, "content": content})
    print("Message posted!")

# main loop
while True:
    print("\n--- Message Board ---")
    print("1. View messages")
    print("2. Post message")
    print("3. Exit")
    
    choice = input("Choose: ")
    
    if choice == "1":
        show_messages()
    elif choice == "2":
        add_message()
    elif choice == "3":
        break
    else:
        print("Invalid choice")